src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java

```diff
@@ -1230,7 +1230,7 @@ protected void setDisabled(boolean disabled) {
      * @return {@code true} if and only if {@link #setDisabled(boolean)} is implemented
      * @since 6.1.0
      */
-    protected boolean supportsMakeDisabled() {
+    public boolean supportsMakeDisabled() {
```

Comment:
Not that it works if we only make the method `ComputedFolder#supportsMakeDisabled` public and keep this one protected but to keep it consistent we should make the parent public too, this would also prevent consumer to make a mistake if they override it.